PurchasesCoreSwift/Attribution/AttributionPoster.swift

```diff
@@ -155,6 +155,7 @@ import Foundation
     }
 
     private func latestNetworkIdAndAdvertisingIdentifierSent(network: AttributionNetwork) -> String? {
+        print(#function)
```

Comment:
this print can probably be deleted

Reply:
I have 2 prs with this, forgot to clean this one up 🤦‍♂️